wercker.yml

```diff
@@ -9,6 +9,7 @@ start-build:
         name: report system
         code: |
           cat /proc/version
+          cat /etc/*release
```

Comment:
builds are failing currently for unknown reasons, but this change isn't responsible for that.